src/web_app/js/call.js

```diff
@@ -419,23 +419,45 @@ Call.prototype.maybeCreatePcClient_ = function() {
   if (this.pcClient_) {
     return;
```

Comment:
Looked more closely, and I agree with your analysis. I don't think there's a simple comment that really adds a lot of value, so I think it's fine as-is.